src/vcProject.cpp

```diff
@@ -833,3 +837,101 @@ void vcProject_ExtractAttributionText(udProjectNode *pFolderNode, const char **p
 
   
 }
+
+void vcProject_UpdateProjectInformationDisplayTextures(vcState *pProgramState)
+{
+  const char *pInfo = nullptr;
+  if (udProjectNode_GetMetadataString(pProgramState->activeProject.pRoot, "information", &pInfo, "") == udE_Success)
+  {
+    if (pProgramState->projectInfoTextures.pLastInfoText != pInfo)
+    {
+      udFree(pProgramState->projectInfoTextures.pLastInfoText);
+      pProgramState->projectInfoTextures.pLastInfoText = udStrdup(pInfo);
+      for (vcTexture *pTexture : pProgramState->projectInfoTextures.textures)
+        vcTexture_Destroy(&pTexture);
+      for (const char *pStr : pProgramState->projectInfoTextures.infoStrings)
+        udFree(pStr);
+      pProgramState->projectInfoTextures.infoStrings.Clear();
+      pProgramState->projectInfoTextures.textures.Clear();
+      pProgramState->projectInfoTextures.textureSizes.Clear();
+
+      const size_t invalidLen = udStrlen(pInfo);
+
+      // Discover Images
+      int firstIndex = 0;
+      int startStrIndex = 0;
+      while (true)
+      {
+        size_t startImagePos = 0;
+        size_t openBracketPos = 0;
+        size_t closeBracketPos = 0;
+        size_t openParenthesisPos = 0;
+        size_t closeParenthesisPos = 0;
+
+        if (udStrchr(pInfo + firstIndex, "!", &startImagePos) == nullptr)
+          break;
+        startImagePos += firstIndex;
+        if (udStrchr(pInfo + startImagePos, "[", &openBracketPos) == nullptr)
+          break;
+        openBracketPos += startImagePos;
+        if (udStrchr(pInfo + openBracketPos, "]", &closeBracketPos) == nullptr)
+          break;
+        closeBracketPos += openBracketPos;
+        if (udStrchr(pInfo + closeBracketPos, "(", &openParenthesisPos) == nullptr)
+          break;
+        openParenthesisPos += closeBracketPos;
+        if (udStrchr(pInfo + openParenthesisPos, ")", &closeParenthesisPos) == nullptr)
+          break;
+        closeParenthesisPos += openParenthesisPos;
+
+        if (startImagePos == invalidLen ||
+          openBracketPos == invalidLen ||
+          closeBracketPos == invalidLen ||
+          openParenthesisPos == invalidLen ||
+          closeParenthesisPos == invalidLen)
+          break;
```

Comment:
This can never be hit - when any of these are `invalidLen`, `udStrchr` will return `nullptr`.